components/examine/CommentsBox.vue

```diff
@@ -1,10 +1,11 @@
+/* eslint-disable vue/no-v-html */
 <template>
   <div class="flex flex-col space-y-2 overflow-auto">
     <div
       class="comments-box-child flex flex-col rounded border border-gray-400 bg-white p-2 text-sm"
       v-for="comment in comments"
     >
-      <p>{{ comment.comment }}</p>
+      <p v-html="comment.html" />
```

Comment:
I don't think we need to use `v-html` here, we can use the whitespace classes in Tailwind https://tailwindcss.com/docs/whitespace
```suggestion
      <p class="whitespace-pre-wrap">{{ comment.comment }}</p>
```
Changes in other files wouldn't be needed either.

Reply:
Not using v-html is certainly safer, since a user could enter code in the comment, right? (Or is it sanitized?)

Reply:
Doesn't seem like the comments are sanitized when you push up a new one, this XSS payload works for example: `<img src="xss-test" onerror="alert(1)">`. However, AFAIK only the examiners can create new comments? I could see it being a problem if the system auto-generates a comment that contains user-generated content in it that's not sanitized.
Regardless, the Tailwind solution is a one-liner and avoids all this stuff.